Rebus.AdoNet/Dialects/PostgreSqlDialect.cs

```diff
@@ -75,7 +75,14 @@ public override string GetColumnType(DbType type, uint length, uint precision, u
 			var result = identity ? GetIdentityTypeFor(type)
 				: base.GetColumnType(type, length, precision, scale, identity, primary);
 
+#if false
+			// XXX: Avoid tow 'PRIMARY KEY' in output string of column:
+			//		CREATE TABLE "sagas" ( "id" uuid PRIMARY KEY  PRIMARY KEY NOT NULL ,...
+			//		GetColumnType.cs/GetColumnType() do this correctly.
+
 			return string.Format("{0} {1}", result, primary ? " PRIMARY KEY" : "");

```

Comment:
The actual 'PRIMARY KEY' string should be provided by GetColumnType / GetIdentityTypeFor calls.